src/actions/index.js

```diff
@@ -1,8 +1,22 @@
 // we'll need axios
+import axios from 'axios';
 
 // we'll need to create 3 different action types here.
 // one for fetching, one for success and one for failure
+export const FETCH_START = 'FETCH_START';
+export const FETCH_SUCCESS = 'FETCH_SUCCESS';
+export const FETCH_FAIL = 'FETCH_FAIL';
```

Comment:
as of now, with only one specific data set we're after, this is fine -- but if there were extra data sets or extra possible fetch actions happening, understand you'd want to be as specific as possible with `CHARS_FETCH_START` or the like